test/integration/test/list.test.js

```diff
@@ -1,17 +1,26 @@
 const clientHelper = require('../src/client-helper');
 const configHelper = require('../src/config-helper');
 const RevAiApiJob = require('../../../dist/src/models/RevAiApiJob')
+const client = clientHelper.getClient(configHelper.getApiKey());
+
+beforeAll(async (done) => {
+    const jobList = await client.getListOfJobs();
+    if(jobList == undefined || jobList.length < 2) {
+        for(var i = 0; i < 2; i++) {
```

Comment:
minor: you could just call `client.submitJobUrl` twice instead of looping. I think it's preference though